stablehlo/reference/Element.cpp

```diff
@@ -184,6 +184,30 @@ Element mapWithUpcastToDouble(const Element &el, FloatFn floatFn,
                                      debugString(type).c_str()));
 }
 
+bool compareEqual(APFloat A, APFloat B) {
```

Comment:
Let's make this name more specific, since "compare" may mean many things for floats. How about following LLVM's nomenclature, i.e. `fcmpOeq`, `fcmpOlt` and `fcmpOgt`, for these three functions?